.github/actions/tag_trigger_action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: tag_trigger_action
+description: Trigger a workflow on a tag addition
+inputs:
+  tag-name:
+    description: 'The tag to trigger the workflow on'
+    required: true
+  test-name:
```

Comment:
I don't see where this input is used?

Reply:
it's used in `... grep ${{ inputs.tag-name }} --quiet;...` 😂